arch/cpu/cc2538/Makefile.cc2538

```diff
@@ -1,7 +1,7 @@
 ifndef SOURCE_LDSCRIPT
 SOURCE_LDSCRIPT = $(CONTIKI_CPU)/cc2538.lds
 endif
-LDSCRIPT = $(OBJECTDIR)/cc2538.ld
+LDSCRIPT ?= $(OBJECTDIR)/cc2538.ld
```

Comment:
This is not required and in fact could potentially cause harm. For this CPU the user can override already using `SOURCE_LDSCRIPT` above.